src/Humanizer.Tests/Humanizer.Tests.csproj

```diff
@@ -24,9 +24,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="ApiApprover\PublicApiApprovalTest.approve_public_api.approved.txt">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    <ContentWithTargetPath Include="ApiApprover\PublicApiApprovalTest.approve_public_api.approved.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <TargetPath>PublicApiApprovalTest.approve_public_api.approved.txt</TargetPath>
```

Comment:
Why did this need to change?

Reply:
The ApiApprover test was failing and it appeared to me that the reason for this was that the approved.txt file was not being copied to the directory where the test runner expected it to be. I made this change, which appeared to solve the problem. However, after reverting the change, the test still passes, so I guess the change was unnecessary. Reverted in 947466a.